src/Fantomas.Core/CodePrinter.fs

```diff
@@ -3084,12 +3084,16 @@ and genTypeDefn (TypeDef (ats, px, leadingKeyword, ao, tds, tcs, equalsRange, td
 
         typeName
         +> opt sepNone impCtor genMemberDefn
-        +> genEq SynTypeDefn_Equals equalsRange
+        +> leadingExpressionIsMultiline (opt sepNone impCtor genMemberDefn) (fun isMulti ctx ->
```

Comment:
all tests work, but are there any other implications about `genEq SynTypeDefn_Equals equalsRange` which I don't see?

Reply:
`genEq` shouldn't really be touched here.
The problem is a bit of a different nature.